pipelines/DECam/ApVerify.yaml

```diff
@@ -15,11 +15,28 @@ imports:
       - afterburner
 tasks:
   # ApVerify override removed by excluding apPipe.
-  diaPipe:
+  associateApdb:
     class: lsst.ap.association.DiaPipelineTask
     config:
+      connections.coaddName: parameters.coaddName
```

Comment:
Shouldn't this connection override (and many others like it, in this and other pipelines) already be inherited from `_ingredients/ApVerify.yaml`?

Reply:
See comment on line 17: the process of splicing in $AP_PIPE_DIR/pipelines/DECam/ApPipe.yaml throws out those changes.

However, I would ask the same question in general -- shouldn't most of these overrides be inherited from (some flavor of) `ApPipe.yaml`?